haystack/document_store/base.py

```diff
@@ -19,9 +19,11 @@ class BaseDocumentStore(BaseComponent):
     index: Optional[str]
     label_index: Optional[str]
     similarity: Optional[str]
+    duplicate_documents_options: tuple = ('skip', 'overwrite', 'fail')
 
     @abstractmethod
-    def write_documents(self, documents: Union[List[dict], List[Document]], index: Optional[str] = None):
+    def write_documents(self, documents: Union[List[dict], List[Document]], index: Optional[str] = None,
+                        batch_size: int = 10_000, duplicate_documents: Optional[str] = None):
```

Comment:
How about this name `duplicate_docs_handling`

Reply:
@lalitpagaria We can do that, but @tholor mentioned it to use that parameter name.

Reply:
`duplicate_documents` is bit generic. Yes agree let's wait for @tholor

Reply:
My reasoning for the shorter 'duplicate_documents': I think you will mostly see / use the argument in combination with one of the values (skip, overwrite, fail) which kinda makes clear that this is some kind of "handling". So I don't see a significant value provided by the additional word. In contrast, it would rather "pollute" our YAML configs of pipelines or user code in scripts.